pages/api/create-new-room.js

```diff
@@ -0,0 +1,29 @@
+import dbConnect from '../../utils/dbConnect';
```

Comment:
good so far

Reply:
might be better to put the file in a /api/spaces directory. I know we don't currently do that now but we should refactor to do that later for the rest of the files later.